src/content/ui-api/kubectl-gs/template-app.md

```diff
@@ -85,3 +85,7 @@ status:
     status: ""
   version: ""
 ```
+
+## Related
+
+- [`kubectl gs login`]({{< relref "/ui-api/kubectl-gs/login" >}}) - Ensure an authenticated kubectl context.
```

Comment:
I don't think `template app` should be related to `login`, as the command doesn't require the user to be logged in